CHANGELOG.md

```diff
@@ -1,222 +1,333 @@
-# Changelog
+# Change Log
```

Comment:
The new format is required to automatically parse the change log and extract the latest changes when creating a production release. You can view the format better here: https://github.com/square/anvil/blob/ralf/publishing/CHANGELOG.md